client/signup/steps/theme-selection/index.jsx

```diff
@@ -30,28 +34,55 @@ module.exports = React.createClass( {
 		};
 	},
 
-	renderThemes: function() {
+	handleScreenshotClick: function( theme ) {
```

Comment:
I don't fully get what you meant. `handlScreenshotClick` is passed as prop to `ThemesList` and forwarded to `Theme` in order to instruct what to do when a screenshot is clicked. In this case this method submits the step.